include/kframework/foundry.md

```diff
@@ -236,6 +242,16 @@ module FOUNDRY-CHEAT-CODES
           <addressSet> .Set </addressSet>
           <storageSlotSet> .Set </storageSlotSet>
         </whitelist>
+        <recordAccess>
+          <isRecordActive> false </isRecordActive>
+          <records>
+            <record multiplicity="*" type="Map">
+              <recordKey> 0 </recordKey>
+              <reads> .List </reads>
+              <writes> .List </writes>
+            </record>
+          </records>
+        </recordAccess>
```

Comment:
A few changes are still required. First of all this configuration needs to be initialized in `kevm-pyk/src/foundry.py` in the `_init_term(` function with
```python
+        'ISRECORDACTIVE_CELL': FALSE,
+        'RECORDS_CELL': KApply('.RecordCellMap'),
```